solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -110,13 +108,15 @@ library DKG {
 
     event DkgStarted(uint256 indexed seed);
 
-    // TODO: Revisit properties returned in this event when working on result
-    // challenges and the client.
-    //  TODO: Should it also return seed to link the result with a DKG run?
     event DkgResultSubmitted(
         bytes32 indexed resultHash,
+        uint256 indexed seed,
+        uint256 submitterMemberIndex,
         bytes indexed groupPubKey,
-        address indexed submitter
+        uint8[] misbehavedMembersIndices,
+        bytes signatures,
+        uint256[] signingMembersIndices,
+        uint32[] members
```

Comment:
Let's also add a note that the final order of members in the group might be different after DKG completes. See `Groups.setGroupMembers`. Clients should `getGroup` after DKG to see what is their final group member ID in that group.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2734